salt/containers/config/usr-local-bin-setup-secrets.sh

```diff
@@ -5,6 +5,9 @@ set -e
 #VAULT_ROLE_ID
 #VAULT_SECRET_ID
 
+# store secret id for future logins
+sudo echo "export VAULT_SECRET_ID=$VAULT_SECRET_ID" > ~/.vault-secret
```

Comment:
do we still need the `sudo` here? who is this script run as?
would it be worth to also change ownership (if necessary) and change the permissions?

(could also just store the secret itself in the file)

Reply:
No, should be removed. The only user we have here is `ubuntu` so it's both the user that runs this script and what Jenkins uses (all managed from there).